cms/envs/aws.py

```diff
@@ -280,6 +280,16 @@
 for database_name in DATABASES:
     DATABASES[database_name]['ATOMIC_REQUESTS'] = True
 
+# Override database settings with environment variables, if any
+DATABASES['default'].update({
```

Comment:
Why this change? Would people using Swift not be able to use `lms.auth.json` or 'lms.env.json`?